### ISSUE TITLE
Pluxee

### ISSUE THREAD

**Issue by ivonulens** [Fllorent0D/ha-custom-component-pluxee-be#5]
Hello

Sodexo ammount is not more showing, is this becouse it changed from Sodexo To Pluxee

**Comment by bugzke** [comment]
Certainly seems that way. This integration scrapes the information from the website, and their website changed.

In the integration code I notice
```
LOGIN_URL = "https://sodexo4you.be/fr"
LUNCH_PASS_SELECTOR = "body > header > div.header-fixed > div.balance-block > div > ul > li.lunch-pass > a > span.balance_price"
ECO_PASS_SELECTOR = "body > header > div.header-fixed > div.balance-block > div > ul > li.eco-pass > a > span.balance_price"
GIFT_PASS_SELECTOR = "body > header > div.header-fixed > div.balance-block > div > ul > li.cadeau-pass > a > span.balance_price"
```

However, when looking at the source code of the new website design, the HTML is different. So probably why it doesn't update anymore.

It looks like this for e.g. the lunch pass:
`body > div > header > div.header-fixed > div.balance-block > div > ul > li > a > span.balance--price"`
![image](https://github.com/Fllorent0D/ha-custom-component-sodexo-be/assets/84719694/2a40d478-de73-4394-9f49-b0111ba4815d)

Not quite sure though how to get the data without the class in the li element indicating what type the balance is referring to.
It's in the a href attribute, but I don't know how this integration should be adjusted to work with that.

@Fllorent0D can you help with this?

**Comment by Tib612** [comment]
Hi, I made a PR: #6 to fix this.
User accessing their data on sodexo4you.be will now be able to access it in HA.
For user wanting to access data on pluxee, you can make a PR on [https://github.com/Tib612/pluxee-api](url)

**Comment by bugzke** [comment]
Imported your PR in my HA and confirmed working. Very nice, thanks a lot!

**Comment by Fllorent0D** [comment]
@Tib612 Thanks for the work !

**Comment by Fllorent0D** [comment]
closed by #6
